app/main.py

```diff
@@ -1 +1,5 @@
-# write your code here
+def copy_file(command: str) -> None:
+    list_file = command.split()
+    if list_file == 3 and list_file[1] != list_file[2]:
+        with open(list_file[1], "r") as f, open(list_file[2], "w") as g:
```

Comment:
Code style issue: According to the checklist, it's better to use descriptive names for file handlers. Instead of 'f' and 'g', you could use 'source_file' and 'destination_file' respectively.